StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -217,47 +213,62 @@ final class LinkAccountPickerViewController: UIViewController {
         self.footerView = footerView
         footerContainerView.addAndPinSubview(footerView)
 
-        bodyView.selectAccount(nil) // activate the logic to list all accounts
+        bodyView.selectAccounts([]) // activate the logic to list all accounts
     }
 
-    private func didSelectConectAccount() {
-        guard let selectedAccountTuple = dataSource.selectedAccountTuple else {
-            assertionFailure("user shouldn't be able to press the connect account button without an account")
+    private func didSelectConnectAccounts() {
+        let nextPane: FinancialConnectionsSessionManifest.NextPane? = {
+            if let mostRecentlySelectedAccount = dataSource.selectedAccounts.last {
```

Comment:
just double checking: the `nextPaneOnSelection` should be the same for each account? so just grabbing the last works out?

Reply:
yes - there's a bit of an assumption because backend controls what is returned (and they can change it) but that should be true

[this is the equivalent web code](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/f87b7a8a/src/linkedAccounts/inner/components/panes/v3/LinkAccountPickerPane/containers/ReturningUser.tsx#L107-L127) where they also pick the last account in the array